CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 **Internal**:
```

Comment:
Could you, please, add `Unreleased` section name?

Reply:
I definitely didn't forget to add this in my last PR...